src/interfaces/IRewardsManager.sol

```diff
@@ -73,18 +64,6 @@ interface IRewardsManager {
     view
     returns (PreviewClaimableRewards[] memory);
 
-  function previewUndrippedRewardsRedemption(uint16 rewardPoolId_, uint256 depositReceiptTokenAmount_)
```

Comment:
We need to make sure there is a way for Dave to calculate APYs since we are deleting this.

Maybe we can add a `previewCurrentUndrippedRewards`?